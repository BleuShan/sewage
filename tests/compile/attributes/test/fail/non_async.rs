#[sewage::test(runtime(tokio))]
fn run() {}

fn main() {}
