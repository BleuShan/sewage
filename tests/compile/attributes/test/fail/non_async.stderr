error: Must be async
 --> $DIR/non_async.rs:2:1
  |
2 | fn run() {}
  | ^^^^^^^^
